test/testTimeSelect.jsx

```diff
@@ -48,66 +44,71 @@ describe('TimeSelect', function() {
   });
 
   describe('render', function() {
-    it('creates a react-bootstrap select input', function() {
-      var renderOutput = shallowRender(<TimeSelect />);
-      assert.equal(renderOutput.type, Input);
-      assert.equal(renderOutput.props.type, 'select');
+    var props;
+    beforeEach(function() {
+      props = {
+        type: 'select',
+        name: 'Time',
+        className: 'input-sm',
+        label: 'Time'
+      };
+    });
+
+    it('should render the reactIntl', function() {
+      var timeSelect = shallow(<TimeSelect />);
+      expect(timeSelect.find(ReactIntl.IntlProvider)).to.have.length(1);
     });
 
     it('can be provided a default value as a Date instance', function() {
       var date = new Date(2015, 1, 1, 15, 30);
-      var renderOutput = shallowRender(<TimeSelect value={date} />);
-      assert.equal(renderOutput.props.value, '15:30');
+      var timeSelect = shallow(<TimeSelect value={date}/>);
+      expect(timeSelect.find(Input).props().value).to.equal('15:30');
     });
 
-    it('passes through name, label and className', function() {
-      var renderOutput = shallowRender(<TimeSelect name="foo" label="bar" className="baz" />);
+    it('should render the Input field', function() {
+      var timeSelect = shallow(<TimeSelect />);
+      expect(timeSelect.find(Input)).to.have.length(1);
+    });
 
-      assert.equal(renderOutput.props.name, 'foo');
-      assert.equal(renderOutput.props.className, 'baz');
-      assert.equal(renderOutput.props.label, 'bar');
+    it('should render the Input field with a type prop', function() {
+      var timeSelect = shallow(<TimeSelect {...props}/>);
+      expect(timeSelect.find(Input).props().type).to.equal(props.type);
+    });
```

Comment:
`type` is statically defined on the input so the way the test is defined is slightly confusing. Could you take `type` out of the props object here and instead do:
```
expect(timeSelect.find(Input).props().type).to.equal('select');
```